validator_client/src/lib.rs

```diff
@@ -208,46 +206,36 @@ impl<T: EthSpec> ProductionValidatorClient<T> {
             fork_service,
             block_service,
             attestation_service,
-            exit_signals: vec![],
             config,
         })
     }
 
     pub fn start_service(&mut self) -> Result<(), String> {
-        let duties_exit = self
+        let _ = self
```

Comment:
confirm all these `let _ = ` are what we want

Reply:
All these functions also return `()`

Reply:
Then none of them need `let _ = `